cmd/storage-rpc-client.go

```diff
@@ -116,8 +116,13 @@ func newStorageRPC(endpoint Endpoint) StorageAPI {
 // Stringer interface compatible representation of network device.
 func (n *networkStorage) String() string {
 	// Remove the storage RPC path prefix, internal paths are meaningless.
-	serviceEndpoint := strings.TrimPrefix(n.rpcClient.ServiceEndpoint(), storageRPCPath)
-	return n.rpcClient.ServerAddr() + ":" + serviceEndpoint
+	serviceEndpoint := strings.TrimPrefix(n.rpcClient.ServiceEndpoint(),
```

Comment:
The path is only canonicalized here. Server doesn't need any change. See this is a String method.